install instructions in INSTALL.md fail

The INSTALL.md file says:

```
To install RaySession, simply run as usual: <br/>
`$ make` <br/>
`$ [sudo] make install`
```

( would you be up to drop those <br/> from that file ? not everyone reads file from the web )

But:
```
[strk@cin:/usr/src/music/raysession/raysession(master)] make
make[1]: Entering directory '/home/src/music/raysession/raysession/HoustonPatchbay'
make[1]: *** No targets specified and no makefile found.  Stop.
```

--- a/INSTALL.md
+++ b/INSTALL.md
@@ -1,45 +1,69 @@
-# ---  INSTALL for RAYSESSION  ---
+# Preparing the source tree
 
-Before installing, please uninstall any existing RaySession installation: <br/>
-`$ [sudo] make uninstall`
+If you just cloned the git repository, make sure you also
+cloned the submodules, which you can do using:
 
-To install RaySession, simply run as usual: <br/>
-`$ make` <br/>
-`$ [sudo] make install`
+    $ git submodule update --init
 
-depending of the distribution you'll need to use LRELEASE variable to install.
-If you don't have 'lrelease' executable but 'lrelease-qt5' use:
-`$ make LRELEASE=lrelease-qt5` <br/>
-`$ [sudo] make install`
+# Building
 
-You can run RaySession without install, by using instead: <br/>
-`$ make` <br/>
-`$ ./src/bin/raysession`
+## Build dependencies
 
-Packagers can make use of the 'PREFIX' and 'DESTDIR' variable during install, like this: <br/>
-`$ make install PREFIX=/usr DESTDIR=./test-dir`
-
-
-
-To uninstall RaySession, run: <br/>
-`$ [sudo] make uninstall`
-<br/>
-
-===== BUILD DEPENDENCIES =====
---------------------------------
-The required build dependencies are: <i>(devel packages of these)</i>
+The required build dependencies are: (devel packages of these)
 
  - PyQt5
- - Qt5 dev tools 
+ - Qt5 dev tools
  - qtchooser
 
-On Debian and Ubuntu, use these commands to install all build dependencies: <br/>
-`$ sudo apt-get install python3-pyqt5 pyqt5-dev-tools qtchooser qttools5-dev-tools`
+On Debian and Ubuntu, use these commands as root to install all build
+dependencies:
 
-===== RUNTIME DEPENDENCIES =====
+    apt-get install \
+      python3-pyqt5 pyqt5-dev-tools \
+      qtchooser qttools5-dev-tools
+
+
+To build RaySession, simply run as usual:
+
+    make
+
+Depending of the distribution you might need to use the LRELEASE variable
+to build.  If you don't have 'lrelease' executable but 'lrelease-qt5' use:
+
+    make LRELEASE=lrelease-qt5
+
+# Installing
+
+To install RaySession, simply run as usual:
+
+    make install
+
+Packagers can make use of the 'PREFIX' and 'DESTDIR' variable during install,
+like this:
+
+    make install PREFIX=/usr DESTDIR=./test-dir
+
+# Uninstalling
+
+To uninstall RaySession, run:
+
+    make uninstall
+
+# Running
+
+You can run RaySession without install, by using:
+
+    ./src/bin/raysession
 
 To run it, you'll additionally need:
 
- - python3-liblo
+   - python3-liblo
 
-IMPORTANT : since python 3.11, because pyliblo has been totally abandonned by Dominic Sacre, you need to use the following fork: https://github.com/gesellkammer/pyliblo3, simply install it with `python3 -m pip install pyliblo3`.
+IMPORTANT: since python 3.11, because pyliblo has been totally abandonned
+by Dominic Sacre, you need to use the following fork:
+
+    https://github.com/gesellkammer/pyliblo3
+
+Simply install it with:
+
+    python3 -m pip install pyliblo3

